### PR TITLE
"e".equals("e") 

### PR DESCRIPTION
**Pull request by WeiChienHsu**
## eeeee 

`"e".equals("e")` -> We eee eeeeeeee "e" then "true".

### eeeeeeeee

```
e
e
e
e
e
e
e
e
e
e
e
e
e
e
e
e
```
***

ee, eeeee? eeeee!! 🚀 

eeeee :)

### CODE REVIEW

File: e.java [hunk 1]
```diff
@@ -1,8 +1,8 @@
 public class e {
 
 	public static void main(String[] args) {
-		while(true) {
-			System.out.print("e");
+		while("e".equals("e") {
```
Comment (LinuxMercedes) [inline e.java]: Ee eeee eeeeeee e `)`?

Comment (WeiChienHsu) [reply e.java]: Eeeeee! Eeee eeeeee eeeee `)` 👍

**Comment by WeiChienHsu** [comment]
@LinuxMercedes  eeeee, eeeee Eeeee 🤙

**Comment by LinuxMercedes** [comment]
E :tada:

**Pull request merged**